### PR TITLE
{tools}[dummy/dummy] EasyBuild v4.1.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Kenneths-MacBook-Pro.local - Darwin UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz, Python 2.7.16
See https://gist.github.com/5823ad761285951e623539e15bf74f67 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/c24120f082ff4de7394d07ec1640a529 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3107.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/93fae468a4a8f04a53d76868504f8023 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2500.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0b56986cd9c564f02d06a0110096ed5f for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/4a0df7316a1673c75d900c643455b557 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/440bef56f6564f841c96284026abbb51 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/669c600dee76bd74150dacdae8dc175c for a full test report.

File: easybuild/easyconfigs/e/EasyBuild/EasyBuild-4.1.2.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+name = 'EasyBuild'
+version = '4.1.2'
+
+homepage = 'https://easybuilders.github.io/easybuild'
+description = """EasyBuild is a software build and installation framework
+ written in Python that allows you to install software in a structured,
+ repeatable and robust way."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```
Comment (lexming) [inline easybuild/easyconfigs/e/EasyBuild/EasyBuild-4.1.2.eb]: should this be replaced with SYSTEM?

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/f82bd71bd6484ad305b6f6f8c2217672 for a full test report.

File: easybuild/easyconfigs/e/EasyBuild/EasyBuild-4.1.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/e/EasyBuild/EasyBuild-4.1.2.eb]: @lexming No, not yet. This is done so people still using EasyBuild 3.x can easily upgrade to EasyBuild 4.x (`SYSTEM` is only known in EasyBuild v3.9.4, not in earlier EasyBuild 3.x releases).

**Pull request merged**